service/src/main/kotlin/io/provenance/api/domain/usecase/cee/ContractUtilities.kt

```diff
@@ -0,0 +1,105 @@
+package io.provenance.api.domain.usecase.cee
```

Comment:
Would it make more sense for this to live in `io.provenance.api.domain.usecase.cee.common`?

Reply:
sure. done